.gitmodules

```diff
@@ -66,3 +62,6 @@
 [submodule "Tools/simulation-ignition"]
 	path = Tools/simulation-ignition
 	url = https://github.com/Auterion/px4-simulation-ignition.git
+[submodule "external/mavlink"]
```

Comment:
Why not keep it under `mavlink/`?

Reply:
At the time I was fighting incremental submodule move non-sense on top the first commit and it was mildly convenient to use a different location (but no real reason).

I think my preference would be we either bury the submodule as close to the actual usage as possible (eg src/modules/mavlink/mavlink) or keep them together, but off to the side like in `external`.

Reply:
`src/modules/mavlink/mavlink` is fine with me as well